### ISSUE TITLE
Instructions Not Clear

### ISSUE THREAD

**Issue by MagicTor** [d0vgan/nppexec#5]
I extracted the files and then moved the NppExec folder into \Notepadd++\Plugins.
There is no .dll files and nothing has changed in notepad++ when I open.

**Comment by d0vgan** [comment]
Are you sure you followed these instructions? --
```
Installation:

Just unpack the NppExec zip-archive to the folder "Notepad++\plugins". It will extract the file "NppExec.dll" as well as the "doc\NppExec" and "NppExec" subfolders (the latter one contains header files used by NppExec at runtime).
```
If it is not clear or needs to be rephrased, could you propose a better wording?

**Comment by d0vgan** [comment]
README.md updated.
ToDo: include the updated README.md into the next release of NppExec.

**Comment by d0vgan** [comment]
Included in NppExec v0.6 RC3.
